src/components/App.tsx

```diff
@@ -3,43 +3,19 @@ import InputItem from './InputItem';
 import ItemList from './ItemList';
 import Footer from './Footer';
 
-export interface StandartItem {
-  text: string;
-  isChecked: boolean;
-  id: number;
-}
-
-const item1: StandartItem = {
-  text: 'дело1',
-  isChecked: false,
-  id: 1,
-};
-
-
-const item2: StandartItem = {
-  text: 'дело2',
-  isChecked: true,
-  id: 2,
-};
-
-const item3: StandartItem = {
-  text: 'дело3',
-  isChecked: false,
-  id: 3,
-};
-
 function App(): JSX.Element {
-  const [items, setItems] = useState<StandartItem[]>([item1, item2, item3]);
-  const amount: number = (items.filter((item: StandartItem) => !item.isChecked)).length;
+  const [checkAll, setCheckAll] = useState(false);
```

Comment:
why it isn't in the redux state?

Reply:
it isn't in the reduction state because less code is required + there is no need to save this state when the application starts